tests/acceptance/02_classes/03_os/ubuntu.cf

```diff
@@ -0,0 +1,51 @@
+#######################################################
+#
+# Test ubuntu classes
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "dummy" string => "dummy";
+}
+
+#######################################################
```

Comment:
I think you don't need this `init` bundle. Maybe not even `test`?

Reply:
I am very unsure about the "best way" to make these tests. I asked nick to review and he approved, so I guess it's okay ;)